regenerate_api.py

```diff
@@ -299,7 +299,7 @@ def main():
 
         shutil.copy(
             './resources/muck-the-duck.svg',
-            os.path.join(STATIC_DIR, 'muck-the-duck.svg'),
+            os.path.join('web/src', 'muck-the-duck.svg'),
```

Comment:
Why the change for this? Is this because of my change?

Reply:
The duck moved from `web/src/static` to` web/src` when we migrated to Vite but haven't needed to regenerate the api since then so was an oversight until now